docs/home/whats-new-2-17.md

```diff
@@ -0,0 +1,145 @@
+
+* <!-- RUN-15878/RUN-16796/RUN-15878/RUN-16085 omit Knative metrics and update supported metrics and hpa autoscaler -->Updated the knative and autoscaler metrics. Run:ai currently supports the following metrics:
+
+    * Throughput
+    * Concurrency
+  
+    For more information, see [Autoscaling metrics](../admin/workloads/inference-overview.md#autoscaling).
+  
+* <!-- RUN-11488/RUN-17720 deprecation of direct metrics in favor of API -->Improved availability of metrics by using Run:ai APIs. Using the API endpoints is more efficient and provides an easier way of retrieving metrics in any application. For more information, see [Metrics](../developer/metrics/metrics.md#changed-metrics-and-api-mapping).
+
+#### Authentication and Authorization
+
+* <!-- RUN-15585/RUN-15586 -  View & Edit SSO settings - SAML -->Added new functionality to SAML 2 in the *SSO configuration* in the *Security* category* of the *Tools & Settings* menu. The added functionality assists with troubleshooting login or permissions issues that may arise. Now administrators now have the ability to:
+
+    * View and edit SSO settings.
+    * Upload or download a settings XML file.
+
+For more information, see [SSO UI configuration](../admin/runai-setup/authentication/sso.md#step-1-ui-configuration).
+
+<!-- #### Policies - 
+no new stories here
+
+### Control and Visibility
+
+no new stories here -->
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform.
+
+### Feature deprecations
+
+Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative. The following features have been marked for deprecation:
+
+### API support and endpoint deprecations
+
+The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. For more information about Run:ai's API support policy, see [Developer overview](../developer/overview-developer.md#overview-developer-documentation).
+
+The following list of API endpoints have been marked for deprecation:
+
+* Jobs&mdash;the *Jobs* feature (submission form and view) has been moved to the category of *Legacy*. To enable them, go to *Tools & Settings*, *General*, open the *Workloads* pane, and then toggle the *Jobs view* and *Job submission* switch to the enabled position.
```

Comment:
@jasonnovichRunAI it seems like this section should be under deprecated features and not deprecated API endpoints, isn't it?